### ISSUE TITLE
JBrowserDriver ignores CSS in Quick-Render-Mode

### ISSUE THREAD

**Issue by Argelbargel** [MachinePublishers/jBrowserDriver#346]
Hi there,

up until 1.0.0RC1 the JBrowserDriver was downloading an parsing CSS-Files embedded via <link>-Tags when Settings#quickRender() was true.

Since 1.0.1 this behaviour was changed and the extension `css` was added to `StreamConnection#mediaExtensions` which defines the extensions of files that are discarded in quick-render mode.

IMHO that is a bug as there are scenarios where one wants to test for correct application of CSS whether a javascript is correctly setting some style-class but still does not want to load all the images and other media embedded in the page.

My concrete use case is testing whether an element isDisplayed() in certain cases which fails when CSS gets discarded.

Currently the only workaround is disabling quick-render-mode.

Kind regards

**Comment by Argelbargel** [comment]
Okay. After taking a look at the ["offending" commit](https://github.com/MachinePublishers/jBrowserDriver/commit/3422103e656e676f266535e022a75253d1d675f5#diff-f2622567b1aeb35da0e47c23f13deec0), i think this works as designed.

Is it correct that to enable the "old" behaviour one has to do this:

```
Capabilities settings = Settings.builder()
                .headless(true)
                .quickRender(false)
                .blockMedia(true)
                .buildCapabilities()
```

Then this can be closed.

**Comment by hollingsworthd** [comment]
Thanks for pointing this out. Yeah I agree with you CSS rendering is too important to have it tied in with these other options. The point of blockMedia and quickRender are to eliminate images/video/etc which rarely are important in headless rendering. Blocking CSS is a rarer use case and should be its own option.

**Comment by hollingsworthd** [comment]
Released in v1.1.0-RC2 available now on Maven Central
